core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -373,6 +378,10 @@ public void start(WorkerNetAddress address) throws IOException {
               Configuration.global(), ServerUserState.global()));
     }
 
+    AlluxioConfiguration conf = Configuration.global();
+    String dbDir = conf.getString(PropertyKey.DORA_WORKER_META_STORE_ROCKSDB_DIR);
```

Comment:
Use Configuration.getString?

Reply:
```
 public static Object get(PropertyKey key, ConfigurationValueOptions options) {
    return SERVER_CONFIG_REFERENCE.get().get(key, options);
  }
...

 public static AlluxioConfiguration global() {
    return SERVER_CONFIG_REFERENCE.get();
  }

```

So, they are the same.
(It is a little confusing because both ways are used in code)
But maybe I will use the `Configuration.getString` because it is used in the same function.